src/js/api/sku.js

```diff
@@ -280,3 +323,46 @@ ripe.Ripe.prototype.validateSkuP = function(id, options) {
         });
     });
 };
+
+/**
+ * Counts the existing SKUs, according to the provided filtering
+ * strategy as normalized values.
+ *
+ * @param {Object} options An object of options to configure the request, such as:
+ * - 'filters[]' - List of filters that the query will use to, operators such as
+ * ('in', 'not_in', 'like', 'contains'), for instance (eg: 'id:eq:42') would filter by the id that equals to 42.

```

Comment:
I don't think we need to be explicit about the way the filters are defined... otherwise, these comments are going to be very difficult to maintain

Reply:
Got it, but this is also in a lot of other places, do we also remove it there?